tests/fixtures/request_payloads/target_namespace.json

```diff
@@ -0,0 +1,19 @@
+{
+    "_COMMENT": [
+        "THIS IS A GENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.",
+        "Instead, update the test tests/unit/test_generate_payload_fixtures.py#test_save_payload_fixtures.",
+        "Run `pytest --update-fixtures` to update it."
+    ],
+    "_id": "64ae4be43a93459ede8c383b",
+    "default_target_id": "64a80107d667dd0c2b13d8cd",
+    "default_version_mode": "AUTO",
+    "dtype": "UNKNOWN",
```

Comment:
Is this fixture get updated? Just to check whether we have remove `UNKNOWN` from default `dtype`.

Reply:
good catch - updated.